ops/global/vault_db.tf

```diff
@@ -8,7 +8,6 @@ resource "azurerm_key_vault" "db_keys" {
   resource_group_name        = data.azurerm_resource_group.rg.name
   sku_name                   = "standard"
   tenant_id                  = data.azurerm_client_config.current.tenant_id
-  soft_delete_enabled        = true
```

Comment:
Why the soft delete removals?

Reply:
Updated the PR root description with a link.

Reply:
Sweet, thanks.